server/utils/startup.ts

```diff
@@ -1,9 +1,35 @@
+import { execSync } from "child_process";
 import chalk from "chalk";
 import env from "@server/env";
 import Logger from "@server/logging/Logger";
 import AuthenticationProvider from "@server/models/AuthenticationProvider";
 import Team from "@server/models/Team";
 
+export function checkPendingMigrations() {
+  const commandResult = execSync("yarn sequelize db:migrate:status");
+  const commandResultArray = Buffer.from(commandResult)
+    .toString("utf-8")
+    .split("\n");
+
+  const pendingMigrations = commandResultArray.filter((line) =>
+    line.startsWith("down")
+  );
+
+  if (pendingMigrations.length) {
+    Logger.warn("You have pending migrations");
+    Logger.warn(
+      pendingMigrations
+        .map((line, i) => `${i + 1}. ${line.replace("down ", "")}`)
+        .join("\n")
+    );
+    Logger.warn(
+      "Please run `yarn sequelize db:migrate` to run all pending migrations"
+    );
```

Comment:
```suggestion
    Logger.warn(
      "Please run `yarn db:migrate` or `yarn db:migrate --env production-ssl-disabled` to run all pending migrations"
    );
```

Reply:
Would be great if we could detect which one of these to print, but it's probably getting too clever

Reply:
Could be possible. I'll let you know if I find a simple(r) solution. For the time being we could put this in a separate `feat` issue.